### DISCUSSION TITLE
Multiple profiles for same application ?

### DISCUSSION THREAD

**Discussion by 6nightrhyme6** [Codectory/AutoActions#142] [General]
**_As there doesn't seem to be any way to Delete my post. I will just excuse myself and say that I just discovered Actions Shortcuts. That solved it. Thanx_**

I have a number of profiles/settings I want to enable at different times for same application, a video player (potplayer).

Profile1: 23.976hz
Profile2: 50hz
Profile3: 23.976hz + HDR
Profile 4: 29,976hz

Is there any way to accomplice this ?

**Comment by TheAndroidx1** [comment]
I would also like an answer, I'm trying to use this to auto switch resolution when launching moonlight games from different devices such as my steam deck or vita.
